src/components/Icon.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable no-unused-vars */
+import React from 'react'
+import styled from 'styled-components'
+
+const IconWrapper = styled.div`
+`
+
+const IconTitle = styled.div`
+  text-transform: uppercase;
+  font-weight: 700;
+  font-family: "Roboto", sans-serif;
+  color: ${(props) => props.color};
+  text-align: center;
+  padding-top: 15px;
+  font-size: 15px;
+  line-height: 1.3;
+`
+
+const IconBorder = styled.div`
+  width: ${(props) => props.size}px;
+  height: ${(props) => props.size}px;
+  border-radius: 50%;
+  border: 2px solid ${(props) => props.color};
+  display: grid;
+  margin: 0 auto;
+`
+
+const IconImage = styled.svg`
+  width: 45%;
+  height: 45%;
+  align-self: center;
+  justify-self: center;
+  fill: ${(props) => props.color};
+`
+
+const variants = {
```

Comment:
Super nice! I have been struggling with the icon component making it reusable not showing text etc.